src/renderer/component/uriIndicator/view.jsx

```diff
@@ -91,6 +91,7 @@ class UriIndicator extends React.PureComponent {
         navigate="/show"
         navigateParams={{ uri: channelLink }}
         className="no-underline"
+        span={span ? span : {}}
```

Comment:
You can just pass this in as is
`span={span}`

If it is `undefined` that is ok